src/aktualizr_lite/test_list.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+set -e
+
+[ -z $DEBUG ] || exec >/tmp/tmp.txt 2>&1
+port="${PORT-9876}"
+
+build_dir=$(pwd)
+src_dir=$1
+
+dest_dir=$(mktemp -d)
+
+cleanup() {
+    echo "cleaning up temp dir"
+    rm -rf "$dest_dir"
+    if [ -n "$pid" ] ; then
+        echo "killing webserver"
+        kill $pid
+    fi
+}
+trap cleanup EXIT
+
+akrepo() {
+    "${build_dir}"/../aktualizr_repo/aktualizr-repo --repotype image --path "$dest_dir" "$@"
+}
+
+add_target() {
+    custom_json="${dest_dir}/custom.json"
+    name=$1
+    sha=$(echo $name | sha256sum | cut -f1 -d\  )
+    cat >$custom_json <<EOF
+{
+  "hardwareIds": ["raspberrypi3-64"],
+  "targetFormat": "OSTREE"
+}
+EOF
+    akrepo --command image \
+        --targetname $name --targetsha256 $sha --targetlength 0 --targetcustom $custom_json
+}
+
+akrepo --command generate --expires 2021-07-04T16:33:27Z
+add_target foo1
+add_target foo2
+akrepo --command signtargets
+
+pushd $dest_dir
+python3 -m http.server $port &
+pid=$!
+sleep 0.1
+
+# curl http://localhost:$port/repo/image/targets.json | json_pp
+
+sysroot=$dest_dir/sysroot
+mkdir $sysroot
+ostree admin init-fs sysroot
+
+sota_dir=$dest_dir/sota
+mkdir $sota_dir
+cat >$sota_dir/sota.toml <<EOF
+[uptane]
+repo_server = "http://localhost:$port/repo/image"
+
+[provision]
+primary_ecu_hardware_id = "raspberrypi3-64"
```

Comment:
I might suggest using an obviously test-only hardware ID just to make it unambiguous that this isn't actually intended to run on an rpi.